deploy/operator-install.yaml

```diff
@@ -174,6 +174,8 @@ spec:
             autoscale:
               description: Autoscale describe autoscaling configuration for the cluster
               properties:
+                disabled:
```

Comment:
Looks like code-gen need to be run again

Reply:
what's missing here? `make codegen` shouldn't be enough?

Reply:
I'm not sure, but some comments can be missed

Reply:
Yes, please run `make codegen` again

Reply:
`make codegen` has nothing more to add. seems we're ok